add link to new documentation

shinobu opened the pull request:
This pull request adds a link to the new documentation

shinobu committed:
add link to new documentation and travis

white-gecko commented:
The README ist still important, so the phrase "This documentation …" is not good. It should just be a prominent link to the documentation. I've already disabled the wiki here on GitHub,

shinobu commented:
oh sorry, i didn't know it was already deactivated, gonna change that in a sec

white-gecko commented:
I think the link should be more prominent. Maybe you add an extra section before the Introduction.

shinobu commented:
you can check it again @white-gecko

white-gecko commented:
The documentation link should go to the top of the README file.
And please squash the commits.

shinobu commented:
I added the link at the top, put the ##Documentation at the bottom (the notice about how they can propose changes is in my point of view important)

white-gecko commented:
Please check it again! Build-Status is now linked to docs.ontowiki.net. You could also update the build status using the travis icons.

shinobu commented:
I have literary no idea how that even could have happened.
Fixed it, added the Travis build status (still kept the owdev one for now), changed the look so that it resembles Erfurt

should be ok now @white-gecko

pfrischmuth commented on README.md:
```diff
@@ -1,6 +1,12 @@
+| Travis CI      | [![Travis CI Build Status](https://travis-ci.org/AKSW/OntoWiki.svg)](https://travis-ci.org/AKSW/OntoWiki/)           |
+| owdev Jenkins  | [![owdev Build Status](http://owdev.ontowiki.net/job/OntoWiki/badge/icon)](http://owdev.ontowiki.net/job/OntoWiki/)  |
+
+[Documentation](https://github.com/AKSW/docs.ontowiki.net)
```
Link seems to be wrong

pfrischmuth commented on README.md:
```diff
@@ -1,6 +1,12 @@
+| CI System      | Status                                                                                                           | 
+| -------------- | ---------------------------------------------------------------------------------------------------------------- |
+| Travis CI      | [![Travis CI Build Status](https://travis-ci.org/AKSW/OntoWiki.svg)](https://travis-ci.org/AKSW/OntoWiki/)           |
+| owdev Jenkins  | [![owdev Build Status](http://owdev.ontowiki.net/job/OntoWiki/badge/icon)](http://owdev.ontowiki.net/job/OntoWiki/)  |
```
Please remove it

white-gecko commented:
I'd like to care for this ;-)

[![Travis CI Build Status](https://travis-ci.org/AKSW/OntoWiki.svg)](https://travis-ci.org/AKSW/OntoWiki/) [![Latest Stable Version](https://poser.pugx.org/aksw/ontowiki/v/stable)](https://packagist.org/packages/aksw/ontowiki) [![Total Downloads](https://poser.pugx.org/aksw/ontowiki/downloads)](https://packagist.org/packages/aksw/ontowiki) [![License](https://poser.pugx.org/aksw/ontowiki/license)](https://packagist.org/packages/aksw/ontowiki) [![composer.lock](https://poser.pugx.org/aksw/ontowiki/composerlock)](https://packagist.org/packages/aksw/ontowiki)


[![OntoWiki User Documentation](https://cdn.rawgit.com/AKSW/OntoWiki/feature/documentationLinkInReadme/application/logo/user_doc.svg)](https://docs.ontowiki.net) [![OntoWiki API Documentation](https://cdn.rawgit.com/AKSW/OntoWiki/feature/documentationLinkInReadme/application/logo/api_doc.svg)](http://api.ontowiki.net/)

white-gecko committed:
Add poser badges

white-gecko committed:
Update image links to develop branch

white-gecko commented:
The actual images can be seen in my comment

pfrischmuth reviewed (approved):
LGTM

white-gecko commented:
@pfrischmuth works, it was the rawgit cache delaying the delivery of the SVG files.
